packages/quantic/force-app/examples/main/lwc/exampleQuanticCaseClassification/exampleQuanticCaseClassification.js

```diff
@@ -60,11 +61,25 @@ export default class ExampleQuanticCaseClassification extends LightningElement {
       description: 'The maximum number of suggestions to be displayed',
       defaultValue: 3,
     },
+    {
+      attribute: 'fetchClassificationOnChange',
+      label: 'Fetch classifications on field value change',
+      description:
+        'Whether the QuanticCaseClassification component should automatically fetch new case classifications when the input field values changes',
+      defaultValue: false,
+    },
+    {
+      attribute: 'fetchDocumentSuggestionOnChange',
+      label: 'Fetch document suggestion on field value change',
+      description:
+        'Whether the QuanticCaseClassification component should automatically fetch new document suggestions when the input field values changes',
```

Comment:
we should update the description, it's not about the input value like previously discussed.